File: samples/python/extra_input.py
"""
Adds two numbers.  Bug: asks for a third number the example never provides.
autotester should say: input mismatch.

Example execution 1:
First: -> 1
Second: -> 2
Sum: 3
"""
a = int(input("First: "))
b = int(input("Second: "))
c = int(input("Third: "))
print(f"Sum: {a + b}")

File: samples/other-languages/extra_input.c
/*
Bug: asks for a third number the example never provides.
autotester should say: input mismatch.

Example execution 1:
First: -> 1
Second: -> 2
Sum: 3
*/
#include <stdio.h>
int main(void) {
    int a, b, c;
    printf("First: ");
    scanf("%d", &a);
    printf("Second: ");
    scanf("%d", &b);
    printf("Third: ");
    scanf("%d", &c);
    printf("Sum: %d\n", a + b);
    return 0;
}
